cliboa/scenario/transform/csv.py

```diff
@@ -30,6 +30,48 @@
 from cliboa.util.string import StringUtil
 from datetime import datetime
 
+import hashlib
+
+
+class CsvColumnHash(FileBaseTransform):
+    """
+    Hash(SHA256) csv (tsv) date field columns
```

Comment:
I think.
This class that hashes the contents of the specified column.
"date field columns" → "specified columns"

Reply:
@nissy0409240 
Thank you for the review.
Modify with above comment.